pkgs/development/ocaml-modules/callipyge/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildDunePackage
+, fetchFromGitHub
+
+, alcotest
+, eqaf
+, fmt
+}:
+
+buildDunePackage rec {
+  pname = "callipyge";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "oklm-wth";
+    repo = "Callipyge";
+    rev = "v${version}";
+    sha256 = "sha256-Xfx96s2MwnfojaF0cWTxgeJeDZNJ93hfv/rhvBQMycU=";
+  };
+
+  useDune2 = true;
+
+  minimumOCamlVersion = "4.05";
+
+  propagatedBuildInputs = [ fmt eqaf ];
+
+  doCheck = true;
+  checkInputs = [ alcotest ];
+
+  meta = {
+    homepage = "https://github.com/oklm-wsh/Callipyge";
```

Comment:
i think the homepage should be `https://github.com/oklm-wsh/callipyge` instead of `https://github.com/oklm-wsh/Callipyge`